src/js/utils/background.js

```diff
@@ -28,6 +28,51 @@ export const normalizeBackground = (background, theme) => {
   return result;
 };
 
+const normalizeBackgroundImage = (background, theme) => {
+  let result;
+
+  if (background.image) {
```

Comment:
```suggestion
  if (background?.image) {
```
is there any chance background could be undefined?

Reply:
I think the check on L165 handles the `undefined` case.

Reply:
Perfect, I agree.